code/parse/sexp.cpp

```diff
@@ -1968,6 +1963,33 @@ int check_sexp_syntax(int node, int return_type, int recursive, int *bad_node, i
 			continue; 
 		}
 
+		// if this is the special argument, make sure it is being properly used
+		if (type2 == SEXP_ATOM_STRING && Sexp_nodes[node].flags & SNF_SPECIAL_ARG_IN_NODE) {
```

Comment:
Just wondering why we're not using `special_argument_appears_in_sexp_list()` here, like we did in the code that's being removed?

Reply:
See item 1 in the PR description... `special_argument_appears_in_sexp_list` checks the entire parameter list.  We only want to do argument syntax checking on the argument node.